react/tic-tac-toe/src/app/screens/Game/components/Board/components/Row/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {indexes} from '../../constants';
+import Square from './components/Square';
+import styles from './components/Square/styles.module.scss';
+
+
+function RenderRow ({ squares, onClick, rowIndex }) {
+  return (
+    <div className={styles.bordRow}>
+      { indexes.map(i =>
```

Comment:
If the prop indexes is undefined `.map` break. I'd add `indexes &&`

Reply:
its a constant, sir. you cant say that